pyomo/core/expr/numvalue.py

```diff
@@ -18,13 +18,20 @@
 
 from pyomo.common.dependencies import numpy as np, numpy_available
 from pyomo.common.deprecation import deprecated
-from pyomo.core.expr.expr_common import \
-    (_add, _sub, _mul, _div, _pow,
-     _neg, _abs, _radd,
-     _rsub, _rmul, _rdiv, _rpow,
-     _iadd, _isub, _imul, _idiv,
-     _ipow, _lt, _le, _eq)
-
+from pyomo.core.expr.expr_common import (
+    _add, _sub, _mul, _div, _pow,
+    _neg, _abs, _radd,
+    _rsub, _rmul, _rdiv, _rpow,
+    _iadd, _isub, _imul, _idiv,
+    _ipow, _lt, _le, _eq
+)
+# TODO: update imports of these objects to pull from numeric_types
```

Comment:
Re the `TODO`, do you (or someone to whom you delegate the task...) plan to update the imports in a different PR?

Reply:
The rework of imports is being deferred "to a later time" - it will probably happen when we finally clean up the expression imports to remove the vestiges of the old system for stitching expression systems (i.e., most of the 'current' module).